src/Files.App/Views/PaneHolderPage.xaml

```diff
@@ -70,84 +57,34 @@
 			NavParams="{x:Bind NavParamsLeft, Mode=OneWay}"
 			PaneHolder="{x:Bind}" />
 
-		<!--  Pane Right  -->
-		<Border
-			x:Name="PaneRightBorder"
-			Grid.Column="1"
-			x:Load="{x:Bind IsRightPaneVisible, Mode=OneWay}">
-			<shellpages:ModernShellPage
-				x:Name="PaneRight"
-				ContentChanged="Pane_ContentChanged"
-				Loaded="Pane_Loaded"
-				NavParams="{x:Bind NavParamsRight, Mode=OneWay}"
-				PaneHolder="{x:Bind}" />
-		</Border>
-
-		<!--  Pane Resizing Splitter  -->
+		<!--  Sizer  -->
 		<toolkit:GridSplitter
 			x:Name="PaneResizer"
 			Grid.Column="1"
-			Width="5"
-			MinWidth="5"
-			HorizontalAlignment="Left"
+			Width="4"
+			x:Load="{x:Bind IsRightPaneVisible, Mode=OneWay}"
 			Background="Transparent"
 			Canvas.ZIndex="150"
 			DoubleTapped="PaneResizer_OnDoubleTapped"
-			Foreground="Transparent"
-			GripperForeground="Transparent"
 			IsTabStop="False"
 			Loaded="PaneResizer_Loaded"
 			ManipulationCompleted="PaneResizer_ManipulationCompleted"
 			ManipulationStarted="PaneResizer_ManipulationStarted"
+			Opacity="0"
 			ResizeBehavior="BasedOnAlignment"
-			ResizeDirection="Auto"
-			Style="{StaticResource DefaultGridSplitterStyle}">
-			<toolkit:GridSplitter.RenderTransform>
-				<TranslateTransform X="0" />
-			</toolkit:GridSplitter.RenderTransform>
-		</toolkit:GridSplitter>
+			Style="{StaticResource DefaultGridSplitterStyle}" />
+
+		<!--  Pane Right  -->
+		<shellpages:ModernShellPage
```

Comment:
I thought this needed to be wrapped in a border for something. Perhaps @gave92 will know.

Reply:
For now reverted.
I thought itd be ok because when I saw your change on this this left in your single commit without discussion.
https://github.com/files-community/Files/pull/7824/commits/26957954d6ce4a99ea6d93916c06cdbad068b8f8#diff-532bef17cf7fcb9657770727df869172f428d84c811de9524fe79dccedda6fdcL74-R80

Reply:
I tried removing it once before but we had to add it back for some reason.

Reply:
> For now reverted.

Did you push this yet?

Reply:
I thought I did, thank you that you did it for me!

Reply:
I removed the background but I didn't do anything for this.

Reply:
I'm not sure if we need to revert it, everything seems to be working properly.